FILE .tool-versions
RANGE {"path":".tool-versions","old_lines":[1,2],"new_lines":[1,4],"diff_block_lines":[5,11],"hunk_header":"@@ -1,2 +1,4 @@","substantive_changed_lines":6}
@@ -1,2 +1,4 @@
-rust 1.85.1
-nodejs 22.20.0
+rust        1.85.1
+nodejs      22.20.0
+java        21
+android-sdk latest
